### PR TITLE
Fix default data description

### PR DESCRIPTION
**Pull request by XX-Yin**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- If computer maintainers need to manually update anything, provide detailed step by step instructions
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
Added the "add_default_project_name" to the settings csv to enable us to turn off the default setting for the project name when the mouse ID is not in the planning spreadsheet. 

### What issues or discussions does this update address?
resolved https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/770

### Describe the expected change in behavior from the perspective of the experimenter
Please add `"add_default_project_name":false` to the settings csv if you have your own project information when running mice not in the behavior rig. 

### Describe any manual update steps for task computers
No

### Was this update tested in 446/447?
Tested on my own computer

### CODE REVIEW

File: src/foraging_gui/Foraging.py [hunk 1]
```diff
@@ -3917,8 +3918,8 @@ def _Start(self):
                     self.Metadata_dialog.ProjectName.setCurrentIndex(index)
                     self.Metadata_dialog._show_project_info()
                     logging.info('Setting Project name: {}'.format(project_name))
-                    add_default = False
-            if add_default:
+                    self.add_default_project_name = False
+            if self.add_default_project_name:
```
Comment (alexpiet) [inline src/foraging_gui/Foraging.py:3922]: This logic seems wrong. We only want to add the default project name if `self.add_default_project_name` is True, AND the (project_name is not None).

So I would add back `add_default`, and then change the if block to `if self.add_default_project_name and add_default`

File: src/foraging_gui/Foraging.py [hunk 2]
```diff
@@ -3918,27 +3941,11 @@ def _Start(self):
...
-                        'Grant Number': ['nan'],
-                        'Investigators': ['Jeremiah Cohen'],
-                        'Fundee': ['nan'],
-                    }
-                    self.Metadata_dialog.project_info = project_info
-                    self.Metadata_dialog.ProjectName.addItems([project_name])
-            self.project_name = project_name
 
+            if self.add_default_project_name and add_default:
+                self._set_default_project()
```
Comment (alexpiet) [inline src/foraging_gui/Foraging.py:3946]: It should be `project_name = self._set_default_project()`, right?
And then that function should return `project_name`?

Comment (XX-Yin) [reply src/foraging_gui/Foraging.py]: that's right! Updated

**Comment by alexpiet** [comment]
Can you merge into "develop", instead of main?

File: src/setting_templates/ForagingSettings.json [hunk 3]
```diff
@@ -18,5 +18,6 @@
     "go_cue_decibel_box1":73,
     "go_cue_decibel_box2":73,
     "go_cue_decibel_box3":73,
-    "go_cue_decibel_box4":73
+    "go_cue_decibel_box4":73,
+    "add_default_project_name":false
```
Comment (alexpiet) [inline src/setting_templates/ForagingSettings.json:22]: I would set the default in the settings_template to make the default in the code (true, not false)

Comment (XX-Yin) [reply src/setting_templates/ForagingSettings.json]: changed

**Review by alexpiet — APPROVED** [review]
Looks great, thanks for taking care of this

**Pull request merged**